boot/freeldr/fdebug/lang/ro-RO.rc

```diff
@@ -50,36 +56,36 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Opțiuni de conectare"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Alegeți portul COM (ex. COM1):", IDC_STATIC, 7, 7, 108, 8
+    LTEXT "Treceți portul COM (ex. COM1):", IDC_STATIC, 7, 7, 108, 8
     EDITTEXT IDC_COMPORT, 7, 17, 182, 14, ES_AUTOHSCROLL
-    LTEXT "Rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
+    LTEXT "Treceți rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
     EDITTEXT IDC_BAUTRATE, 7, 48, 182, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", IDOK, 45, 79, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 100, 79, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 45, 79, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 100, 79, 50, 14
 END
 
 IDD_CAPTURE DIALOGEX 0, 0, 251, 95
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Fișierul de captură"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Numele fișierului:", IDC_STATIC, 7, 17, 62, 8
+    LTEXT "Nume fișier capt.:", IDC_STATIC, 7, 17, 62, 8
     EDITTEXT IDC_CAPTUREFILENAME, 7, 26, 181, 14, ES_AUTOHSCROLL
-    PUSHBUTTON "&Caută", IDC_BROWSE, 194, 26, 50, 14
-    DEFPUSHBUTTON "Con&firmă", IDOK, 139, 74, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 194, 74, 50, 14
+    PUSHBUTTON "&Specificare", IDC_BROWSE, 194, 26, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 139, 74, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 194, 74, 50, 14
 END
 
 /* String Tables */
 
 STRINGTABLE
 BEGIN
-    IDS_APP_TITLE "Program de depanare FreeLoader"
-    IDS_HELLO "Salut, lume!"
+    IDS_APP_TITLE "Depanatorul FreeLoader"
+    IDS_HELLO "Salut lume!"
     IDC_FDEBUG "FDEBUG"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_LICENSE "Această aplicație este un soft public; poate fi redistribuită și/sau modificată respectând termenii Licenței Publice Generale GNU publicată de Free Software Foundation; sau respectând Licența în versiunea 2, sau (la alegere) o altă versiune ulterioară.\r\n\r\nAceastă aplicație este distribuită în speranța că va fi utilă, FĂRĂ însă NICI O GARANŢIE; nici măcar cu garanția implicită a VANDABILITĂŢII sau a UTILITĂŢII ÎNTR-UN SCOP ANUME.  Consultați Licența Publică Generală GNU pentru mai multe detalii.\r\n\r\nPuteți vedea această licență aici:\r\nhttp://www.gnu.org/licenses/gpl.html\r\n\r\nDe asemenea puteți consulta traduceri neoficiale ale acestei licențe aici:\r\nhttp://www.gnu.org/licenses/translations.html"
```

Comment:
Don't we have an official (or so-so) original Romanian translation of the GNU GPL clause? This seems to be the one https://staff.cs.upt.ro/~gnu/Licenta_GPL-3-0_RO.html (you have to scroll down to `Cum să aplicaţi aceşti termeni la noul dumneavoastră Program`)

I'd rather use that than figuring out how to translate the clause ourselves.

Reply:
Done!